modules/gdscript/gdscript_tokenizer.h

```diff
@@ -117,11 +111,11 @@ class GDScriptTokenizer {
 			BREAKPOINT,
 			CLASS,
 			CLASS_NAME,
-			CONST,
+			TK_CONST, // Conflict with WinAPI.
```

Comment:
We also had `TK_EOF` prefixed similarly for similar reasons.

We may want to go all the way and rename all with a `TK_` prefix as some of those are very generic names, like `IS` or `CLASS` or `ENUM` which can be prone to conflict.